controlpanel/api/aws.py

```diff
@@ -96,14 +96,14 @@ def iam_assume_role_principal():
     "Version": "2012-10-17",
     "Statement": [
         {
-            "Sid": "console",
+            "Sid": "ListUserBuckets",
             "Action": [
-                "s3:GetBucketLocation",
                 "s3:ListAllMyBuckets",
-                "s3:ListBucketVersions",
```

Comment:
Are we sure `s3:GetBucketLocation` and `s3:ListBucketVersions` are not used/required for certain operations? If they're not used than good to go 👍

Reply:
both of these have been moved to the LIST_ACTIONS block 👍